07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,45 @@
+section {
+display: flex;
+padding-top: 15px;
+width: 850px;
+margin-top: 20px;
+background-color: #D3D3D3;
+}
+
+
+header {
+    background: url(../img/nasa-background.jpg);
+    height: 450px;
+    width: 850px;
+    margin-bottom: 5px;
+}
+
+header h1 {
+  color: white;
+  font-size: 72px;
+  font-family: sans-serif;
+  text-align: center;
+  padding-top: 215px;
+
+}
+
+.box1 {
```

Comment:
I showed how to line up the boxes with both the header and nav using flexbox, I don't see that you went back to fix this. Why?